modules/sql-engine/src/testFixtures/java/org/apache/ignite/internal/sql/engine/util/tpch/TpchHelper.java

```diff
@@ -65,6 +65,32 @@ public static String getQuery(String queryId) {
         }
     }
 
+    /**
+     * Loads query plan for provided TPC-H query id.
+     *
+     * @see #getQuery(String) for query id details.
+     */
+    public static String getQueryPlan(String queryId) {
```

Comment:
it's better to move this as utility method to the test class. The problem is plan belongs to a test resource of sql-engine test jar, not test-fixture jar. Thus, in order to get plan, you have to add test to a dependency alongside with reps to text-fixture, otherwise you will get `java.lang.IllegalArgumentException: Resource does not exist` exception